src/components/atoms/ActionButton.js

```diff
@@ -44,4 +47,12 @@ ActionButton.propTypes = {
    * Callback for when a user's mouse leaves the button
    */
   onMouseLeave: PropTypes.func,
+  /**
+   * Optional overrides the default css
+   */
+  customClass: PropTypes.string,
+  /**
+   * Control Id to help testng
+   */
+  id: PropTypes.string,
```

Comment:
this is confusing, there could be a case for setting an id for the button. rename this prop to dataCy